homepage.css

```diff
@@ -0,0 +1,66 @@
+body{
+  font-family: 'Josefin Sans', sans-serif;
+  text-align:center;
+}
+
+.button{
+  background-color:white;
+  height:40px;
+  width:200px;
+  border-radius:30px;
+  color:#6C6C6C;
+  text-align:center;
+  padding-top:22px;
+  paddinf-bottom:14px;
```

Comment:
There's a typo in your rule here - if it's not working like you expect, this is why!